src/Sarif/Visitors/UpdateIndicesVisitor.cs

```diff
@@ -9,12 +9,12 @@ namespace Microsoft.CodeAnalysis.Sarif.Visitors
     public class UpdateIndicesVisitor : SarifRewritingVisitor
     {
         private IDictionary<string, int> _fullyQualifiedLogicalNameToIndexMap;
-        private IDictionary<string, int> _fileLocationKeyToIndexMap;
+        private IDictionary<FileLocation, int> _fileLocationToIndexMap;
 
-        public UpdateIndicesVisitor(IDictionary<string, int> fullyQualifiedLogicalNameToIndexMap, IDictionary<string, int> fileLocationKeyToIndexMap)
+        public UpdateIndicesVisitor(IDictionary<string, int> fullyQualifiedLogicalNameToIndexMap, IDictionary<FileLocation, int> fileLocationToIndexMap)
```

Comment:
>fileLocationToIndexMap [](start = 129, length = 22)

The one place this class is instantiated in product code, you now set this parameter to `null`. Is this a work in progress? Because as the product code stands in this branch, `VisitFileLocation` never does anything.

Reply:
Yes. The VisitFileLocation in this visitor only applies to result matching. So this functionality is still pending. The prerelease transformer doesn't need this update to function.

---
In reply to: [247716347](https://github.com/Microsoft/sarif-sdk/pull/1212#discussion_r247716347) [](ancestors = 247716347)